GetQueueSpidMessages/function.json

```
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "queueName": "spid-msg-items",
```

```suggestion
      "queueName": "spidmsgitems",
```

should contain only letters: https://github.com/search?q=org%3Apagopa+queueName&type=Code